XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs

```diff
@@ -194,7 +194,7 @@ private void ExtractFile()
             }
             catch (Exception e)
             {
-                log.ErrorFormat("Exception occurred when extracting downloaded archive: {0}", e.Message);
+                log.InfoFormat("Exception occurred when extracting downloaded archive: {0}", e.Message);

```

Comment:
I'm not sure about this change; maybe we should keep it at error level because, if we do a text search in a big log file for errors, it might be overlooked.

Reply:
I'm not sure either, I will revert it